frontend/src/components/provider/providersTable.tsx

```diff
@@ -29,13 +35,21 @@ export function ProvidersTable({ providers }: Props) {
     }),
     columnHelper.accessor("created_at", {
       header: () => <span>Created at</span>,
+      cell: (info) => convertTimestamp(info.getValue()!),
     }),
   ];
-  const table = useReactTable<Provider>({
+  const table = useReactTable<ProviderPublic>({
     data: providers,
     columns: columns,
     getCoreRowModel: getCoreRowModel(),
   });
 
-  return <DefaultTableRender<Provider> table={table} />;
+  return (
+    <DefaultTableRender<ProviderPublic>
+      table={table}
+      rowHook={(order) => {
+        router.navigate({ to: `/providers/${order.id}` });
```

Comment:
Do we want here to redirect to order_id? Not provider_id? 
Check if that order or provider exists first